ci/python-app.yml

```diff
@@ -19,10 +19,10 @@ jobs:
 
     steps:
     - uses: actions/checkout@v4
-    - name: Set up Python 3.10
-      uses: actions/setup-python@v3
+    - name: Set up Python 3.13
+      uses: actions/setup-python@v5
       with:
-        python-version: "3.10"
+        python-version: "3.13"
```

Comment:
LGTM! with Conditions:

 * verified improvement of `actions/setup-python@v5` (from `v3`).
 * Meta-data is consistent and matches the Python version used.
   <details><summary>NIT</summary>

     > [!TIP]
     > knit-picky or out-of-scope:
     > Consider improving with a single variable to set the python version and
     > access it for both name and python-version via GHA contexts.
     > See [Documentation](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs) for details.)

   </details>
   <details><summary>NIT - Consider Python `3.12` might be more appropriate than `3.13` for GHA</summary>

     > [!TIP]
     > knit-picky or out-of-scope:
     > Looks like version 3.12 will be the cached version (while `3.13` **is** the
     > latest it will probably be slower for some GitHub Actions)
     > see [https://github.com/actions/runner-images/issues/10636](https://github.com/actions/runner-images/issues/10636)
     > and note the cached versions for python.
     >
     > Should still work though; Just slightly slower.

   </details>

**Hope this helps** :bow:

Reply:
Regarding the variables:
I've checked some other files. Have not seen that it was done like this somehwere. For example the android, ant and maven templates are using the version number in the step name and for the version requirement. But always explicitly written. So yes, technically a considerable idea, but have not seen that it is done somewhere else like this.

Using Python 3.12 instead of 3.13 makes sense and I have changed.